components/brave_extension/extension/brave_extension/components/advancedView/index.tsx

```diff
@@ -128,6 +130,9 @@ export default class Shields extends React.PureComponent<Props, State> {
                 httpUpgradableResources={shieldsPanelTabData.httpUpgradableResources}
                 httpsRedirectedResources={shieldsPanelTabData.httpsRedirectedResources}
                 httpsEverywhereToggled={actions.httpsEverywhereToggled}
+                // SpeedReader controls
+                speedreader={shieldsPanelTabData.speedreader}
```

Comment:
this is more of a product issue than a sec issue, but @tomlowenthal expressed concerns with showing the speedreader toggle in shields (if that is what this is doing)